src/main/resources/view/EventListCard.fxml

```diff
@@ -30,7 +30,8 @@
       <Label fx:id="date" styleClass="cell_small_label" text="\$date" />
       <Label fx:id="loc" styleClass="cell_small_label" text="\$location" />
       <Label fx:id="time" styleClass="cell_small_label" text="\$time" />
-         <Label fx:id="description" styleClass="cell_small_label" text="\$description" />
+      <Label fx:id="description" styleClass="cell_small_label" text="\$description" />
```

Comment:
Might need to set wrap text to true, otherwise if the description or list of volunteers is too long it might cause some display issues?

Reply:
<img width="700" alt="image" src="https://github.com/user-attachments/assets/411e4057-cb45-4884-ad70-c90ddcb3796a">
Okie, I have added wrapText='true' and this is how it wraps now